### PR TITLE
Enable azure-blob-storage in the DEB and RPM packages

### PR DESCRIPTION
**Pull request by tobim**
This adds support for the `azure-blob-storage` operator to all packages built on the static binary build.

### CODE REVIEW

File: libtenzir/CMakeLists.txt [hunk 1]
```diff
@@ -354,6 +361,11 @@ dependency_summary("libmaxminddb" maxminddb::maxminddb "Dependencies")
 
 # Link against Apache Arrow.
 target_link_libraries(libtenzir PUBLIC ${ARROW_LIBRARY})
+if (TENZIR_ENABLE_STATIC_EXECUTABLE)
+  target_link_libraries(
+    libtenzir PUBLIC Azure::azure-identity Azure::azure-storage-blobs
+                     Azure::azure-storage-files-datalake)
+endif ()
```
Comment (tobim) [inline libtenzir/CMakeLists.txt:368]:
```suggestion
```

**Review by IyeOnline — APPROVED** [review]
Seems Fine To Me

File: libtenzir/CMakeLists.txt
Comment (dominiklohmann) [inline libtenzir/CMakeLists.txt:1]: This should probably get a changelog entry stating that the azure-blob-storage connector is now also available in the static binary.

Comment (dominiklohmann) [reply libtenzir/CMakeLists.txt:1]: I'll add this in a separate PR.

**Pull request merged**